content/en/docs/alphashape.md

```diff
@@ -6,28 +6,118 @@ sections:
 weight: 40
 ---
 
-GOAT allows you to calculate and visualize isochrones using alpha shapes. GOAT is using a very fast 2D concave hull algorithm, which is described in this paper  [A New Concave Hull Algorithm and Concaveness Measure for n-dimensional Datasets, 2012](https://journal.iis.sinica.edu.tw/paper/1/100295-3.pdf?cd=2217EEBB7C44EDA26) and is implemented in a Javascript library called [Concaveman](https://github.com/mapbox/concaveman). This library generates isochrones (polygons) representing the area from a set of points that can be reached in a dedicated time.
+#### Isochrone Definition
 
-![](/images/docs/technical_documentation/alphashape/isochrone_as_alphashape.webp)
+An isochrone or travel time map is an indicator that provides users with a clear visualization of how far they can travel in a given time from a selected location. Isochrones can be calculated from one or more starting points for modes such as walking, biking, driving, or public transport. Each mode includes input factors such as speed, infrastructure network, and travel time.
 
-#### 1. Concave Hull
-Concave hulls are often used to generalize bounding polygons around a given sets of points. Depending on the chosen concaveness the precision of the isochrone can differ. The following example illustrates the difference between a convex and a concave hull. 
+In GOAT, Isochrone shapes are dynamically created in the front end based on a travel time grid. Therefore, isochrone can be created fast and for different intervals on the, it allows to dynamically compute isochrone shapes and boundaries from the road network for each step in the front end.
```

Comment:
"and for different intervals" word is missing here. I believe it should be on the fly. 

maybe like this: "and for different intervals on the fly, which allows to dynamically compute isochrone shapes and boundaries from the road network for each step.